workloads/network-perf-v2/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+source ./env.sh
+source ../../utils/common.sh
+
+curl -sS -L $NETPERF_URL | tar -xz
+
+# Assuming kubeconfig is set
+oc create ns netperf
+oc create sa netperf -n netperf
+oc adm policy add-scc-to-user hostnetwork -z netperf -n netperf
+
+log "###############################################"
+log "Workload: ${WORKLOAD}"
+log "###############################################"
+
+./k8s-netperf --debug --all --config ${WORKLOAD} --search $ES_SERVER --clean=true
```

Comment:
Do we want to enable debug by default?

Reply:
e2e is primarily used for CI, IMHO so having more output the better. k8s-netperf also doesn't generate a significant amount of logs. 

fine to have a switch, I just didn't think it was necessary.